### PR TITLE
Fix LCD output and add translations

### PR DESCRIPTION
**Pull request by 3d-gussner**

### CODE REVIEW

**Comment by 3d-gussner** [comment]
Please verify and approve the new translations. Remember that the message is limited to 20 characters and 3 rows,
- [x] :czech_republic:  @DRracer
- [X] :de: @3d-gussner
- [x] :es: 
- [x] :fr: @Ilovemyhous @ErwanAliasr1
- [x] :it: @wavexx
- [x] :poland: @arekm
- [x] :netherlands: @vintagepc @stelgenhof
- [x] :romania: @leptun @Hauzman
- [x] :hungary: @AttilaSVK @Hauzman
- [ ] :croatia: @Prime1910 @PRPA041
- [x] :slovakia: @ingbrzy @shatter136
- [ ] :sweden: @Painkiller56
- [ ] :norway: @OS-kar @trondkla

**Review by ErwanAliasr1 — APPROVED** [review]
FR is :ok:

**Review by arekm — APPROVED** [review]
PL is OK.

**Comment by Ro3Deee** [comment]
@leptun, for Romanian:
Instead "Nu există filament încărcat" I'd use "Filamentul nu este încărcat" sau "Filamentul nu este detectat"

File: lang/po/Firmware_sk.po [hunk 1]
```diff
@@ -2597,6 +2597,16 @@ msgstr "Tryska je horúca! Počkajte na vychladnutie."
 msgid "Nozzle changed?"
 msgstr "Vymenili ste trysku?"
 
+#. MSG_MISSING_FILAMENT_CONTINUE c=20 r=3
+#: ../../Firmware/messages.cpp:157 ../../Firmware/util.cpp:388
+msgid "There is no filament loaded. Continue?"
+msgstr "Nie je vložené žiadne vlákno. Pokračovať?"
```
Comment (ingbrzy) [inline lang/po/Firmware_sk.po]: "Nie je zavedený žiaden filament. Pokračovať?"

File: lang/po/Firmware_sk.po [hunk 2]
```diff
@@ -2597,6 +2597,16 @@ msgstr "Tryska je horúca! Počkajte na vychladnutie."
...
+
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Nie je vložené žiadne vlákno. Tlač zrušená."
```
Comment (ingbrzy) [inline lang/po/Firmware_sk.po]: "Nie je zavedený žiaden filament. Tlač zrušená."

**Comment by 3d-gussner** [comment]
> @leptun, for Romanian: Instead "Nu există filament încărcat" I'd use "Filamentul nu este încărcat" sau "Filamentul nu este detectat"

Thanks @petrubecheru @Hauzman. Which is the preferred one?
1. Filamentul nu este încărcat :tada: 
2. Filamentul nu este detectat :heart:

File: lang/po/Firmware_nl.po [hunk 3]
```diff
@@ -2624,6 +2617,16 @@ msgstr "Mondstuk is heet! Wacht op afkoeling."
...
 
+#. MSG_MISSING_FILAMENT_CONTINUE c=20 r=3
+#: ../../Firmware/messages.cpp:157 ../../Firmware/util.cpp:388
+msgid "There is no filament loaded. Continue?"
+msgstr "Geen filament geladen. Doorgaan?"
+
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Geen filament geladen. Afdrukken geannuleerd."
```
Comment (stelgenhof) [inline lang/po/Firmware_nl.po]: Although 'afdrukken' is the correct translation for 'to print', in the context of 3D printing it is a bit strange. I suggest to keep the English verb "to print" as that is more commonly used in the community.

So the translation should then be "Geen filament geladen. Printen geannuleerd."

With that said, other translations in this file should be adjusted accordingly for consistency's sake.

**Review by stelgenhof — CHANGES_REQUESTED** [review]

**Review by gudnimg — APPROVED** [review]
Looks good to me. :) I’ll wait a bit for translators to review before merging.

**Comment by AttilaSVK** [comment]
Hungarian should be: "Nincs befűzve filament. Folytassam?" and "Nincs befűzve filament. Nyomtatás megállítva."

File: lang/po/Firmware_sk.po (hunk 1 above)
Comment (ingbrzy) [reply lang/po/Firmware_sk.po]: @3d-gussner can you update it?
Thank you

File: lang/po/Firmware_cs.po [hunk 4]
```diff
@@ -2599,6 +2592,16 @@ msgstr "Tryska je horká! Počkejte na vychladnutí."
 msgid "Nozzle changed?"
 msgstr "Vyměnili jste trysku?"
 
+#. MSG_MISSING_FILAMENT_CONTINUE c=20 r=3
+#: ../../Firmware/messages.cpp:157 ../../Firmware/util.cpp:388
+msgid "There is no filament loaded. Continue?"
+msgstr "Není vloženo žádné vlákno. Pokračovat?"
```
Comment (DRracer) [inline lang/po/Firmware_cs.po]: Není vložen filament. Pokračovat?

File: lang/po/Firmware_cs.po [hunk 5]
```diff
@@ -2599,6 +2592,16 @@ msgstr "Tryska je horká! Počkejte na vychladnutí."
...
+
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Není vloženo žádné vlákno. Tisk zrušen."
```
Comment (DRracer) [inline lang/po/Firmware_cs.po]: Není vložen filament. Tisk zrušen.

File: lang/po/Firmware_it.po [hunk 6]
```diff
@@ -2622,6 +2615,16 @@ msgstr "L'ugello è caldo! Attendere il raffreddamento."
...
+#. MSG_MISSING_FILAMENT_CONTINUE c=20 r=3
+#: ../../Firmware/messages.cpp:157 ../../Firmware/util.cpp:388
+msgid "There is no filament loaded. Continue?"
+msgstr "Non è caricato alcun filamento. Continuare?"
+
+#. MSG_MISSING_FILAMENT_CANCELLED c=20 r=8
+#: ../../Firmware/messages.cpp:158 ../../Firmware/util.cpp:389
+msgid "There is no filament loaded. Print cancelled."
+msgstr "Non è caricato alcun filamento. Stampa annullata."
+
```
Comment (wavexx) [inline lang/po/Firmware_it.po:2627]: Looking good! You could marginally shorten the message with `Nessun filamento caricato.` as the first sentence if that's beneficial.

**Comment by 3d-gussner** [comment]
Hi
:croatia: @Prime1910 @PRPA041
:slovakia: @ingbrzy @shatter136
:sweden: @Painkiller56
:norway: @OS-kar @trondkla

Can you please let me know if you will be able to review the changes until end of this week? Also please let me know if you are still interested in reviewing the community translations. If not, I will not ask you anymore and will NOT translate new messages anymore or use "faulty" computer based translations instead.

**Review by shatter136 — COMMENTED** [review]
Hi. Sorry. In _sk, I would rather write "Nie je vložený filament. ... "
And please keep us still in the loop. Thank you.

**Review by ingbrzy — CHANGES_REQUESTED** [review]

**Pull request merged**